pkg/processor/runtime/rpc/abstract.go

```diff
@@ -439,6 +439,7 @@ func (r *AbstractRuntime) watchWrapperProcess() {
 		return
 	}
 
+	// NOTE: this error message is filtered when reading logs (on shouldAddToBriefErrorsMessage())- consider that when changing it!
 	r.Logger.ErrorWith("Unexpected termination of child process",
```

Comment:
Instead make this string a const in common and reuse it here and there to enforce match